SignList.xml

```diff
@@ -0,0 +1,16 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <FirstParty Include="Xamarin.Firebase.*.dll" />
+    <FirstParty Include="Xamarin.Google.*.dll" />
```

Comment:
Are there other prefixes that we want to sign?

Reply:
Seems like the actual .dll patterns are:

 - `Firebase.*.dll`
 - `Google.*.dll`

https://github.com/xamarin/GoogleApisForiOSComponents/search?q=%3CAssemblyName%3E